### PR TITLE
[Libs] LG support for Email/Calendar/Todo/POI skill libs part

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
#2058 Support LG in Email/Calendar/Todo/POI skill.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
upgrade bot framework libs to 4.6.0-Daily-2019-07-31-01.
update test adapter to mock auth.

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
Yes

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
release a package that skills will reference.

#### General
- [x] I have commented my code, particularly in hard-to-understand areas	
- [x] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

**Review by ryanisgrig — APPROVED** [review]

File: lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Testing/DefaultTestAdapter.cs [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+        private const string conversationId = "conversation";
+        private const string conversationName = "Conversation";
+        private const string serviceUrl = "https://test.com";
+        private const string oauthConnection = "Azure Active Directory";
+
+        public DefaultTestAdapter(BotStateSet botStateSet, string connectionName = oauthConnection, string token = oauthConnection)
             : base(sendTraceActivity: false)
         {
             Use(new EventDebuggerMiddleware());
+
```
Comment (lauren-mills) [inline lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Testing/DefaultTestAdapter.cs:23]: Are these changes required for LG?

**Review by lauren-mills — CHANGES_REQUESTED** [review]

File: lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Testing/DefaultTestAdapter.cs (hunk 1 above)
Comment (KayMKM) [reply lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Testing/DefaultTestAdapter.cs:23]: yes. after updated to latest sdk package, need these changes to pass auth in unit test cases

**Review by lzc850612 — APPROVED** [review]

**Review by lauren-mills — APPROVED** [review]
